content/docs/hooks-overview.md

```diff
@@ -1,107 +1,107 @@
+Anda tentunya pernah melakukan pengambilan data, proses berlangganan atau menerima data secara terus menerus (*subscriptions*), atau secara manual merubah DOM dari komponen React sebelumnya. Kami menyebut operasi-operasi tersebut "*side effects*" atau "efek samping" karena dapat mempengaruhi komponen lain dan tidak dapat dilakukan pada saat proses *render*.
 
-The Effect Hook, `useEffect`, adds the ability to perform side effects from a function component. It serves the same purpose as `componentDidMount`, `componentDidUpdate`, and `componentWillUnmount` in React classes, but unified into a single API. (We'll show examples comparing `useEffect` to these methods in [Using the Effect Hook](/docs/hooks-effect.html).)
+Hook Effect, `useEffect`, menambahkan kemampuan untuk melakukan "efek samping" dari sebuah komponen fungsi. Hal tersebut memiliki tujuan yang sama seperti `componentDidMount`, `componentDidUpdate`, dan `componentWillUnmount` pada kelas React, tetapi bersatu menjadi satu API. (Kami akan memberi beberapa contoh perbandingan `useEffect` dengan *method* pada [Menggunakan Hook Effect](/docs/hooks-effect.html).)
 
-For example, this component sets the document title after React updates the DOM:
+Sebagai contoh, komponen berikut menetapkan judul dokumen setelah React memperbarui DOM:
 
 ```js{1,6-10}
 import React, { useState, useEffect } from 'react';
 
 function Example() {
   const [count, setCount] = useState(0);
 
-  // Similar to componentDidMount and componentDidUpdate:
+  // Sama seperti componentDidMount dan componentDidUpdate:
   useEffect(() => {
-    // Update the document title using the browser API
-    document.title = `You clicked ${count} times`;
+    // Memperbarui judul dokumen menggunakan API browser
+    document.title = `Anda klik sebanyak ${count} kali`;
   });
 
   return (
     <div>
-      <p>You clicked {count} times</p>
+      <p>Anda klik sebanyak {count} kali</p>
       <button onClick={() => setCount(count + 1)}>
-        Click me
+        Klik saya
       </button>
     </div>
   );
 }
 ```
 
-When you call `useEffect`, you're telling React to run your "effect" function after flushing changes to the DOM. Effects are declared inside the component so they have access to its props and state. By default, React runs the effects after every render -- *including* the first render. (We'll talk more about how this compares to class lifecycles in [Using the Effect Hook](/docs/hooks-effect.html).)
+Pada saat memanggil `useEffect`, Anda memerintah React untuk menjalankan fungsi "efek" setelah membersihkan perubahan pada DOM. Efek di deklarasikan dalam komponen, jadi akan dapat akses pada *prop* dan *state* komponen tersebut. Secara standar, React menjalankan efek setelah setiap *render* -- termasuk pada *render* pertama. (Akan kami bahas lebih banyak mengenai bagaimana hal ini di bandingkan dengan siklus kelas pada [Menggunakan Hook Effect](/docs/hooks-effect.html).)
```

Comment:
```suggestion
Pada saat memanggil `useEffect`, Anda memerintah React untuk menjalankan fungsi "efek" setelah membersihkan perubahan pada DOM. Efek dideklarasikan di dalam komponen sehingga mereka akan mendapat akses pada *prop* dan *state* komponen tersebut. Secara bawaan, React menjalankan efek setelah setiap *render* -- termasuk pada *render* pertama. (Akan kami bahas lebih banyak mengenai bagaimana hal ini jika di bandingkan dengan *lifecycle* pada kelas React di [Menggunakan Hook Effect](/docs/hooks-effect.html).)
```

What do you think? 😄

Reply:
I really should learn Indonesian again 🤦‍♂️